packages/iTwinUI-react/src/core/Select/Select.test.tsx

```diff
@@ -230,12 +231,14 @@ it.each(['Enter', ' ', 'Spacebar'])(
     let menu = document.querySelector('.iui-menu') as HTMLUListElement;
     expect(menu).toBeFalsy();
 
-    fireEvent.keyDown(
-      select.querySelector('.iui-select-button') as HTMLElement,
-      {
-        key,
-      },
-    );
+    act(() => {
```

Comment:
Still don't get when `act` is needed and when it is not 😀

Reply:
when you get warning, it is needed :D